services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusMigrationServiceITest.kt

```diff
@@ -181,6 +180,22 @@ class HakemusMigrationServiceITest(
                 prop(CableReportApplicationData::representativeWithContacts).isNull()
             }
         }
+
+        @Test
+        fun `doesn't change anything when called a second time on the same hanke`() {
+            val (hanke, _) = setup()
+            hakemusMigrationService.migrateOneHanke(hanke.id)
+            val kayttajatAfterFirst = hankeKayttajaService.getKayttajatByHankeId(hanke.id)
+            val applicationId = hakemusRepository.findAll().single().id
+            // Hakemus contains yhteystiedot and yhteyshenkilot
+            val hakemusAfterFirst = hakemusService.getById(applicationId)
+
+            hakemusMigrationService.migrateOneHanke(hanke.id)
+
+            assertThat(hankeKayttajaService.getKayttajatByHankeId(hanke.id))
+                .isEqualTo(kayttajatAfterFirst)
+            assertThat(hakemusService.getById(applicationId)).isEqualTo(hakemusAfterFirst)
```

Comment:
In order to really verify that calling the migrate function again does not do anything, these asserts should also be above after the first call?

Reply:
I think my train of thought got a little sidetracked. Ignore what I said.